.bazelrc

```diff
@@ -23,3 +23,5 @@ build:android --fat_apk_cpu=x86,x86_64
 build:device --ios_multi_cpus=armv7,arm64 --fat_apk_cpu=armeabi-v7a,arm64-v8a
 
 build:fat --ios_multi_cpus=i386,x86_64,armv7,arm64 --fat_apk_cpu=x86,x86_64,armeabi-v7a,arm64-v8a
+
+build:sizeopt -c opt --copt -Os
```

Comment:
Instead of duplicating this from upstream envoy, you should be able to import that `.bazelrc` file like I'm doing here: https://github.com/lyft/envoy-mobile/pull/155/files#diff-f55b2f358053ad76fb5ac3f776f78ef8R1

Reply:
Neat, didn't notice that in your PR.